frontend/lib/ui/ui.types.d.ts

```diff
@@ -3,6 +3,7 @@ export type Variant = "alert" | "default" | "ghost" | "secondary";
 export type AlertType = "error" | "info" | "notify" | "success" | "warning";
 export type FeedbackId = "feedback-error" | "feedback-warning" | "feedback-server-error";
 export type MenuStatus = "accept" | "active" | "empty" | "error" | "idle" | "unsaved" | "warning";
+export type FormSectionType = "errors" | "warnings";
```

Comment:
I think `ValidationResultType` would be a better name, this has nothing to do with form sections

Reply:
Changed in 95c0a724d4bdee6d2378f280652c4269f72b896a I actually had it like that and then changed it again haha